<launch>
    <arg name="uav_id_"/>
    <arg name = "init_x_"/>
    <arg name = "init_y_"/>
    <arg name  = "init_z_"/>

    <node pkg="multi_model_state" name="uav_$(arg uav_id_)_model_state" type="multi_model_state" output="screen">
        <!-- <remap from="/extract_state_client/iris" to="/uav$(arg uav_id)_vins_estimator/odometry"/> -->
         <remap from="~odom" to="/uav_$(arg uav_id_)_visual_slam/odom"/>
         <remap from="~local_odom" to="/uav_$(arg uav_id_)_orb_slam"/>
         <remap from="~global_odom" to="/uav_$(arg uav_id_)_global_odom"/>    
         <param name="multi_model_state/init_x" value="$(arg init_x_)"/>
        <param name="multi_model_state/init_y" value="$(arg init_y_)"/>
        <param name="multi_model_state/init_z" value="$(arg init_z_)"/>
       <param name="multi_model_state/map_frame_id" value="map_uav_$(arg uav_id_)"/>
       <param name="multi_model_state/base_link_frame_id" value="base_link_uav_$(arg uav_id_)"/>    
    </node>
</launch>